frontend/src/views/CensusDirectory/index.tsx

```diff
@@ -17,92 +40,196 @@ function CensusDirectory() {
   `;
 
   const Header = styled.h1`
+    ${fontHeaderXxl}
     margin-bottom: 8px;
     font-weight: 700;
-    ${fontHeaderXxl}
   `;
 
   const Paragraph = styled.p`
-    font-weight: 400;
     ${fontBodyS}
+    font-weight: 400;
```

Comment:
`${fontWeight?.regular}`

This should be updated everywhere in place of `400`.